cli/operator/node.go

```diff
@@ -214,7 +214,9 @@ var StartNodeCmd = &cobra.Command{
 		cfg.SSVOptions.ValidatorController = validatorCtrl
 		cfg.SSVOptions.Metrics = metricsReporter
 
-		operatorNode = operator.New(logger, cfg.SSVOptions, slotTicker)
+		operatorNode = operator.New(logger, cfg.SSVOptions, func() slotticker.SlotTicker {
```

Comment:
What do you think of extracting the anonymous function to a variable (e.g. `slotTickerProvider`)? That would make places where it's passed easier to read, remove code duplication and the variable name would show what's the purpose of this function